data/drak missions.txt

```diff
@@ -36,6 +36,7 @@ mission "Drak guarding Sayaiban"
 		system Sayaiban
 		ship "Archon" "Watchful Gravetender"
 	on enter Sayaiban
+		log "Explored an uninhabited system deep in Korath space, and encountered an Archon who spoke inside your head. Seems this Sayaiban system is home to a 'swarm intelligence' the Korath created. Best steer clear of here."
```

Comment:
I haven't totally decided on what the style of the log entries should be, but I think it makes the most sense for them to be phrased as if written by the pilot: not "you did this," but "(I) did this," with the "I" usually avoided by using sentence fragments.

Also, since this is American English, double quotes should be used rather than single quotes. (Which, in this case, requires using backticks rather than double quotes to enclose the text.)